packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml

```diff
@@ -5,22 +5,23 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
-  name: tkg-aws-default
```

Comment:
modify it in overlay.yaml should be an option as well ?

Reply:
Yes but overlay is usually for more complicated cases